docs/source/conf.py

```diff
@@ -63,15 +63,15 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
```

Comment:
`master_doc` [is a configuration parameter](https://www.sphinx-doc.org/en/1.8/usage/configuration.html#confval-master_doc) in sphinx  so we probably shouldn't change it

Reply:
Fixed